src/main/java/seedu/duke/Duke.java

```diff
@@ -20,9 +21,34 @@
     private Timetable timetable;
     private TagManager tagManager;
 
+    // Character code adapted from http://patorjk.com/software/taag/#p=display&f=Ghost&t=NotUS
+    // Slight modifications made to make it easier on the eyes
+    private static final String NOTUS_LOGO = Formatter.LS
+            + Formatter.LS
+            + "     .-') _               .-') _                 .-')    "
+            + Formatter.LS
+            + "    ( OO ) )             (  OO) )               ( OO ).  "
+            + Formatter.LS
+            + ",--./ ,--,'  .-'),-----. /     '._ ,--. ,--.   (_)---\\_) "
+            + Formatter.LS
+            + "|   \\ |  |\\ ( OO'  .-.  '|'--...__)|  | |  |   /    _ |  "
+            + Formatter.LS
+            + "|    \\|  | )/   |  | |  |'--.  .--'|  | | .-') \\  :` `.  "
+            + Formatter.LS
+            + "|  .     |/ \\_) |  | |  |   |  |   |  | |( OO ) '..`''.) "
+            + Formatter.LS
+            + "|  |\\    |    \\ |  | |  |   |  |   |  | | `-' /.-._)   \\ "
+            + Formatter.LS
+            + "|  | \\   |     `'  '-'  '   |  |  ('  '-'(_.-' \\       / "
+            + Formatter.LS
+            + "`--'  `--'       `-----'    `--'    `-----'     `-----'  "
+            + Formatter.LS;
+
     private static final String WELCOME_MSG_STRING = "Welcome to NotUS! "
+            + NOTUS_LOGO + Formatter.LS
```

Comment:
Can move this over to formatter class, and have a static function to get this.

Reply:
Will do so with my next PR!